### DISCUSSION TITLE
Coinbase wallet not seeing the walletconnect modal

### DISCUSSION THREAD

**Discussion by rahulnrt** [WalletConnect/walletconnect-monorepo#3877] [Developer Support] [answered]
Dear team,

When I copied the Coinbase wallet id from Wallet Connect Explorer and past it my code where I could not see the Coinbase wallet in this modal kidly did the needful

**Comment by glitch-txs** [comment, marked as answer]
Hi, WalletConnectModal doesn't support Coinbase, only Web3Modal SDK, because Coinbase doesn't support the WalletConnect protocol so we need a special connector for them.
